bin/e2e-test-client/src/tests/script.rs

```diff
@@ -67,7 +69,29 @@ pub async fn dry_run(ctx: &TestContext) -> Result<(), Failed> {
     )
     .await??;
 
-    _dry_runs(ctx, &transaction, 1000, DryRunResult::Successful).await
+    _dry_runs(ctx, &[transaction], 1000, DryRunResult::Successful).await
+}
+
+// Dry run multiple transactions
+pub async fn dry_run_multiple_txs(ctx: &TestContext) -> Result<(), Failed> {
+    let transaction1 = tokio::time::timeout(
+        ctx.config.sync_timeout(),
+        ctx.alice.transfer_tx(ctx.bob.address, 0, None),
+    )
+    .await??;
+    let transaction2 = tokio::time::timeout(
+        ctx.config.sync_timeout(),
+        ctx.alice.transfer_tx(ctx.alice.address, 0, None),
```

Comment:
Idk if it matters, but did you mean to transfer from alice to alice?

Reply:
yeah, I had to make the two transactions different otherwise they would have the same Id and this would cause an error.  I'm not sure if there's a way to add a salt.  I also didn't know if these accounts actually had funds, but if they do we could use bob again.